warehouse/transformer/internal/utils/utils.go

```diff
@@ -176,7 +176,13 @@ func IsBlank(value interface{}) bool {
 	case map[string]any:
 		return len(v) == 0
```

Comment:
@achettyiitr Any magic checks for map in js transformer? Like empty string as key and len(v) > 0?

Reply:
No, it's not required. [_.toString()](https://lodash.com/docs/4.17.15#toString) returns something for an empty map as well.